File: app/src/main/java/com/quranapp/android/interfaceUtils/readerIndex/FragReaderIndexCallback.java
/*
 * (c) Faisal Khan. Created on 19/2/2022.
 */

package com.quranapp.android.interfaceUtils.readerIndex;

import android.content.Context;

public interface FragReaderIndexCallback {
    void scrollToTop(boolean smooth);

    void sort(Context ctx);
}

File: app/src/main/java/com/quranapp/android/interfaceUtils/readerIndex/FragReaderIndexCallback.kt
/*
 * (c) Faisal Khan. Created on 19/2/2022.
 */
package com.quranapp.android.interfaceUtils.readerIndex

import android.content.Context

interface FragReaderIndexCallback {
    fun scrollToTop(smooth: Boolean)
    fun sort(ctx: Context)
}